invokeai/frontend/web/src/features/lora/components/ParamLoraSelect.tsx

```diff
@@ -54,7 +54,7 @@ const ParamLoRASelect = () => {
       });
     });
 
-    return data.sort((a, b) => (a.disabled && !b.disabled ? 1 : -1));
+    return data.sort((a, b) => (a.disabled && !b.disabled ? -1 : 1));
```

Comment:
This can be reverted here. I've deployed a more robust fix in #4001

Reply:
Thanks. Reverted.